modules/custom/d_blog/d_blog.install

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * Implements hook_install().
+ */
+function d_blog_install() {
+  \Drupal::service('entity.definition_update_manager')->applyUpdates();
+
+
+  // Add Translation of pagination  iin blog.
+  try {
+    d_blog_add_translation('NEXT', 'DALEJ', 'pl');
```

Comment:
We should use "Next" instead of "NEXT" and change its case by CSS text-transform: uppercase
Capital letters looks bad in Google SERP